pkg/kubectl/cmd/get.go

```diff
@@ -262,10 +279,13 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 
 	for ix := range objs {
 		var mapping *meta.RESTMapping
+		var original runtime.Object
```

Comment:
@lavalamp all the registered column printers and describers work from internal types.